fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraRelaxedLdpIT.java

```diff
@@ -351,6 +363,108 @@ public void testRoundtripping() throws IOException {
+            assertFalse(graph.contains(ANY, binaryNode, RDF.type.asNode(), DIRECT_CONTAINER.asNode()));
+            assertFalse(graph.contains(ANY, binaryNode, CREATED_DATE.asNode(),
+                    createLiteral(newDate, XSDDateTimeType.XSDdateTime)));
+        }
+    }
+
+    @Test
+    public void testChangeInteractionModelPatchRdf() throws Exception {
+        final var postBinary = postObjMethod();
+        postBinary.setHeader(CONTENT_TYPE, "text/plain");
+        postBinary.setEntity(new StringEntity("some test data"));
+        final String binaryUri;
+        // Create a binary
+        try (final var response = execute(postBinary)) {
+            assertEquals(CREATED.getStatusCode(), getStatus(response));
+            binaryUri = getLocation(response);
+        }
+        final Node binaryNode = createURI(binaryUri);
+        final String oldCreatedDate;
+        // Get the binary description
+        try (final var dataset = getDataset(new HttpGet(binaryUri + "/" + FCR_METADATA))) {
+            final var graph = dataset.asDatasetGraph();
+            // Assert it has a NonRDFSource type
+            assertTrue(graph.contains(ANY, binaryNode, RDF.type.asNode(), NON_RDF_SOURCE.asNode()));
+            final var iter = graph.find(ANY, binaryNode, CREATED_DATE.asNode(), ANY);
+            assertTrue(iter.hasNext());
+            final var stmt = iter.next();
+            // We'll change the createdDate
+            oldCreatedDate = ((XSDDateTime) stmt.getObject().getLiteralValue()).toString();
+        }
+        // Now PUT this new graph back
+        final var newDate = ISO_INSTANT.format(Instant.now().atZone(ZoneOffset.UTC));
+        final var patchBinaryDescription = new HttpPatch(binaryUri + "/" + FCR_METADATA);
+        patchBinaryDescription.setHeader(CONTENT_TYPE, "application/sparql-update");
+        patchBinaryDescription.setEntity(new StringEntity("DELETE { <> <" + RDF.type + "> ?type ; " +
+                "<" + CREATED_DATE + "> ?date . } INSERT { <> <" + RDF.type + "> <" + DIRECT_CONTAINER + "> ; " +
+                "<" + CREATED_DATE + "> \"" + newDate + "\"^^<" +
+                XSDDateTimeType.XSDdateTime.getURI() + "> . } WHERE { <> <" + RDF.type + "> ?type ; " +
+                "<" + CREATED_DATE + "> ?date . } "));
+        assertEquals(CONFLICT.getStatusCode(), getStatus(patchBinaryDescription));
+        // Check that the date didn't change either.
+        try (final var dataset = getDataset(new HttpGet(binaryUri + "/" + FCR_METADATA))) {
+            final var graph = dataset.asDatasetGraph();
+            // Assert it has a NonRDFSource type
+            assertTrue(graph.contains(ANY, binaryNode, RDF.type.asNode(), NON_RDF_SOURCE.asNode()));
+            assertTrue(graph.contains(ANY, binaryNode, CREATED_DATE.asNode(),
+                    createLiteral(oldCreatedDate, XSDDateTimeType.XSDdateTime)));
+            assertFalse(graph.contains(ANY, binaryNode, CREATED_DATE.asNode(),
```

Comment:
might be safest to verify that there is only one CREATED_DATE, rather than that this specific value isn't present. Just suggesting this since I thought there was another PR recently that was verify timestamps matched within a certain precision range. If that's not a concern, then fine to ignore this

Reply:
You are right there was a problem but that was from creating a date from an Instant and then converting an RDF value into an Instant. It lost some precision. I was going to do CREATED_BY, but without WebAC that triple never exists. 
So if I check my concern is that you shouldn't have updated the CREATED_DATE, but testing for only a single value won't confirm it didn't change. I'll change the new date to waaaaay in the past and then I can do more general comparison.